### ISSUE TITLE
NVDA does not read captions on youtube.com

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#11729]
### Steps to reproduce:
1. open www.youtube.com on microsoft edge
2. open a video that contains captions
3. Turn the captions on by switching to focus mode and pressing the c key or choosing the captions toggle button
4. you will hear NVDA will say "captions on" but nothing will be read aloud
5. Turn on narrator
6. you will hear as the captions come onto the screen, narrator will read them aloud
### Actual behavior:
NVDA does not read the captions
### Expected behavior:
NVDA should read the captions as they appear
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.2
#### Windows version:
windows 10 build 1909
#### Name and version of other software in use when reproducing the issue:
Microsoft edge
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If addons are disabled, is your problem still occuring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no

**Comment by fatihyuksek1** [comment]
Update:
It reads the caption but when a new caption appears it reads the first one that it has encountered over and over again

**Comment by Adriani90** [comment]
Duplicate of #8337. I am closing this issue in favor of that one because #8337 contains more discussion on this issue which is actually occuring in every browser.
